src/ipc/storage-get.ts

```diff
@@ -0,0 +1,22 @@
+import { IpcMain, IpcMainInvokeEvent } from 'electron'
+import { promises as fs } from 'fs'
+import { promisify } from 'util'
+
+import storage from 'electron-json-storage'
+const get = promisify(storage.get)
+
+export const channel = 'storageGet'
+
+interface StringObject {
+  value: string
+}
+
+async function storageGet(key: string): Promise<object> {
```

Comment:
Though it doesn't make a difference here, it's generally better to make this sort of thing generic:

```ts
async function storageGet<T extends object>(key: string): Promise<T> {
```

It allows the caller to avoid an extra cast.